### ISSUE TITLE
We should probably warn people if OS generated files aregetting included in their assets

### ISSUE THREAD

**Issue by dnfield** [flutter/flutter#47366]
See https://github.com/flutter/flutter/issues/46163#issuecomment-565780153 - we probably want to warn people if they're inadvertently including something like `.DS_Store` in their assets.

**Comment by jonahwilliams** [comment]
I think there is an existing directory or file blacklist. Instead of warning we should just exclude them, or exclude all dotfiles by default

**Comment by dnfield** [comment]
I suppose somone could intentially place a dotfile in there right?  I'm not sure excluding it is the right thing to do.  But maybe warn if the file is a dotfile or marked as hidden on Windows/NTFS.

**Comment by dnfield** [comment]
I seriously doubt they'd want to include `.DS_Store` ever though.

**Comment by jonahwilliams** [comment]
Yeah checking for hidden and excluding automatically seems like the most reasonable option. If they want it included, it can be un-hidden.

@kwkr indicated that they were interested in looking at this
